scripts/CollatePortalJavaScriptSource.groovy

```diff
@@ -6,7 +6,7 @@
  *
  */
 
-includeTargets << grailsScript("Init")
+includeTargets << grailsScript("_GrailsInit")
```

Comment:
Mmmmmm. Yep. Definitely.

Grails 3.x includes a minifier as well. Not sure about 2.4.4...